### ISSUE TITLE
Disconnecting an insight makes dimmers go haywire

### ISSUE THREAD

**Issue by meatheadmike** [pywemo/pywemo#121]
This is the symptom I'm having. I'm not sure what the root cause is yet though. Over the holidays I had 5 wemo switches set up that I don't normally have connected. 4 were insights and 1 was an old school plugin wemo. One of the insights was in the garage and had a weak wifi signal. So sometimes it wouldn't show up properly in the official app. Anyways, I've been having a weird problem where dimmers would randomly start erroring out in HA. I suspect the two are related.

Here's an example error:

`Could not update status for Living room lights ('Insight' object has no attribute 'insight')`

"Living room lights" is of course a dimmer and not an insight!

So this would happen every couple of days or so. Not very often. Well, I just took down my Xmas lights and unplugged all the insights. Guess what error showed up right away?

**Comment by sqldiablo** [comment]
Mike, I've got some pywemo and HASS changes queued up that will significantly change rediscovery. Long story short, I'm changing pywemo to let you disable rediscovery, and I'm modifying HASS to do just that and then to handle rediscovery of devices itself. This isn't the long-term solution for pywemo, but I think it'll go a long way to solve the issues in HASS with WeMo devices, and it should buy me some time to get pywemo's rediscovery figured out.

Long story short, I'll let you know once I've got the new version of pywemo cut and the HASS changes in their dev branch so you can help me test it.

**Comment by meatheadmike** [comment]
Sounds good. I'm happy to help test!

**Comment by sqldiablo** [comment]
PR for pywemo changes is #127.

**Comment by sqldiablo** [comment]
@meatheadmike, is this issue resolved at this point, or is this still happening?

**Comment by pavoni** [comment]
@meatheadmike Feel free to re-open if this is still an issue for you.
